### PR TITLE
updating ha document

### PR DESCRIPTION
**Pull request by JosephNewmanTG**
Associated task: [DOC-1959-HAOverviewDocImprovements](https://graphsql.atlassian.net/browse/DOC-1959)

improvements to HA overview page document.

### CODE REVIEW

**Review by chengjie-qin — APPROVED** [review]
LGTM

File: modules/cluster-and-ha-management/pages/ha-overview.adoc [hunk 1]
```diff
@@ -22,12 +22,18 @@ TigerGraph HA provides continuous operation of some but not all services.
...
++
+NOTE: However, new interpreted and any existing queries can still be executed.
+
+* Schema changes are *not* allowed.
+
+* Backup and export operations are *not available* and will be rejected.
+
+* If the primary node is offline, access to Graph Studio is *interrupted*, but resumes once the primary node is back online.
 
 NOTE: As a workaround, if the failed node cannot be recovered (e.g. hardware issue), full operation can be restored temporarily by the xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[removal of the failing nodes].
```
Comment (songting-chen) [inline modules/cluster-and-ha-management/pages/ha-overview.adoc:39]: or replaced by a good one

File: modules/cluster-and-ha-management/pages/ha-overview.adoc [hunk 2]
```diff
@@ -22,12 +22,18 @@ TigerGraph HA provides continuous operation of some but not all services.
...
+
+* New queries *cannot* be installed.
++
+NOTE: However, new interpreted and any existing queries can still be executed.
+
+* Schema changes are *not* allowed.
+
+* Backup and export operations are *not available* and will be rejected.
+
+* If the primary node is offline, access to Graph Studio is *interrupted*, but resumes once the primary node is back online.
```
Comment (songting-chen) [inline modules/cluster-and-ha-management/pages/ha-overview.adoc:37]: Once 3.10 is released, we can remove this item.

Comment (JosephNewmanTG) [reply modules/cluster-and-ha-management/pages/ha-overview.adoc:37]: Agreed. And thank you. I created a new doc ticket to track improvement efforts for the HA overview. that ticket I linked it to the one associated with this PR but here is the link as well. https://graphsql.atlassian.net/browse/DOC-1974

File: modules/cluster-and-ha-management/pages/ha-overview.adoc (hunk 1 above)
Comment (JosephNewmanTG) [reply modules/cluster-and-ha-management/pages/ha-overview.adoc:39]: Thank you Songting. I have updated this the paragraph above this one, for a better flow, when reading it.

"If an HA system is operating with a failed node, unless the node is recovered, xref:tigergraph-server:cluster-and-ha-management:how_to-replace-a-node-in-a-cluster.adoc[replaced], or the system is reconfigured to xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[exclude that node], the following services are limited or unavailable:"

Adding the replacing cluster instructions.

**Pull request merged**